packages/application-shell/src/components/project-switcher/project-switcher.tsx

```diff
@@ -14,7 +14,7 @@ import { reportErrorToSentry } from '@commercetools-frontend/sentry';
 import AccessibleHidden from '@commercetools-uikit/accessible-hidden';
 import SelectInput from '@commercetools-uikit/select-input';
 import { ErrorIcon } from '@commercetools-uikit/icons';
-import { customProperties } from '@commercetools-uikit/design-system';
+import { designTokens } from '@commercetools-uikit/design-system';
```

Comment:
**nit**: since we can use `ui-kit` design tokens and local ones in `app-kit`, we were aliasing them when importing to make it clear where the used tokens come from ([example](https://github.com/commercetools/merchant-center-application-kit/blob/fb4907897356c21e4b24b277db0df740609db870/packages/application-components/src/components/internals/tabular-page.tsx#L4)).

Perhaps we could use the same convention here.

Reply:
The design tokens import from `app-kit` was aliases in the `user-settings-menu.tsx` file. I think it would also be helpful to alias `ui-kit` design tokens import (something like `uiKitDesignTokens`) in that file and also in this one.

Reply:
Alright, I aliased the imports when we import tokens from both sources, otherwise if it's only uikit I'll prefer to keep the normal import. https://github.com/commercetools/merchant-center-application-kit/pull/2917/commits/4442548f4f89a010acf3e749c331464db2c42a01